cloud/scope/powervs_cluster.go

```diff
@@ -1974,8 +1974,9 @@ func (s *PowerVSClusterScope) ReconcileLoadBalancers() (bool, error) {
 				return false, err
 			}
 
-			if requeue := s.checkLoadBalancerStatus(*loadBalancer); requeue {
-				return requeue, nil
+			if isReady := s.checkLoadBalancerStatus(*loadBalancer); !isReady {
+				s.V(3).Info("LoadBalancer is still not Active", "name", *loadBalancer.Name, "state", *loadBalancer.ProvisioningStatus)
```

Comment:
*loadbalancer.Name can be nil when user sets only id. so better avoid it.

Reply:
It's coming from cloud API resp right, would be able to create a LB without name?

Reply:
oh yeah, sorry, I overlooked loadbalancer from
```
for index, loadBalancer := range loadBalancers {
```